### PR TITLE
Remove clear command

### PR DESCRIPTION
**Pull request by burnflare**
As recommended by https://github.com/CS2103AUG2016-W11-C2/main/pull/133#issuecomment-256676846

### CODE REVIEW

**Comment by rachx** [comment]
i think addcommandtest etc tries running clear too. or should we just hide it?

**Comment by burnflare** [comment]
@rachx passing now, please review.

File: src/test/java/guitests/AddCommandTest.java [hunk 1]
```diff
@@ -29,10 +29,6 @@ public void add() {
         assertResultMessage(AddCommand.MESSAGE_DUPLICATE_TASK);
         assertTrue(taskListPanel.isListMatching(currentList));
 
-        //add to empty list
-        commandBox.runCommand("clear");
```
Comment (rachx) [inline src/test/java/guitests/AddCommandTest.java:33]: empty the list using delete multiple?

Comment (burnflare) [reply src/test/java/guitests/AddCommandTest.java:33]: Can do that, what's the range I should delete?

Comment (burnflare) [reply src/test/java/guitests/AddCommandTest.java:33]: It's 1-9!

Comment (burnflare) [reply src/test/java/guitests/AddCommandTest.java:33]: Updated PR!

**Review by rachx — APPROVED** [review]

**Pull request merged**